docs/pages/machine-id/workload-identity/aws-roles-anywhere.mdx

```diff
@@ -0,0 +1,277 @@
+---
+title: Configuring Workload ID and AWS Roles Anywhere
+description: Configuring AWS to accept Workload ID certificates as authentication using AWS Roles Anywhere
+---
+
+Teleport's Workload Identity issues flexible short-lived identities in X.509
+certificates. AWS Roles Anywhere allows you to use these certificates to
+authenticate to AWS services.
+
+This can be useful in cases where a machine needs to securely authenticate with
+AWS services without the use of a long-lived credential. This is because the
+machine can authenticate with Teleport without using any shared secrets by
+using one of our delegated join methods.
+
+This implementation differs from Teleport's Application Access for AWS feature
+in a few ways:
+
+- Requests to AWS are not proxied through Teleport, meaning reduced latency but
```

Comment:
If we're talking about network traffic, we should refer to specific Teleport components instead of "Teleport". This makes it clearer how the feature interacts with cloud-managed Teleport accounts, high availability, etc.

Reply:
e63011e212